pkg/controllers/provisioning/scheduling/metrics.go

```diff
@@ -74,6 +74,7 @@ var (
 		crmetrics.Registry,
 		prometheus.GaugeOpts{
 			Namespace: metrics.Namespace,
+			Subsystem: schedulerSubsystem,
 			Name:      "ignored_pod_count",
```

Comment:
```suggestion
			Name:      "ignored_pods_count",
```

Given that we use `pods` in other metrics, can we also change this to plural for consistency?

Reply:
Absolutely